sql.txt

```diff
@@ -1,15 +1,29 @@
-(1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
-Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
-
+(1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.
+Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below.
+select CustomerName, ContactName, Country
+from Customer
+ORDER BY Country
+LIMIT 10 OFFSET 10;
+	
+/*limit
+offset is simply the number of records you wish to skip before selecting records*/
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
-
+select * from Customer
+where ContactName LIKE 'A%'
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
-
-(4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
+select * from OrderDetails
+where ProductID = 51 AND quantity > 10
+(4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID,
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
-
-(5) Update the two top rows of the Products Table to increase the price by 1.50. (Just specify the new value; do 
+INSERT INTO products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price)
+VALUES (78, 'Queso Oaxaca', 5, 4, '500 g pkgs.', 15);
+INSERT INTO products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price)
+VALUES (79, 'Queso Fresco El Mexicano', 5, 4, '500 g pkgs.', 5);
+INSERT INTO products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price)
+VALUES (80, 'Queso Manchego', 5, 4, '500 g pkgs.', 10);
+(5) Update the two top rows of the Products Table to increase the price by 1.50. (Just specify the new value; do
 not try to get SQL to do the addition for you.)  Paste your SQL statement below
-
-(6) Delete all rows of the Products Table where the price is less than 7.00. Paste your SQL statement below.
+UPDATE Products
+SET  = price + (1.50)
+WHERE ProductID BETWEEN 1 and 2
```

Comment:
Is this exercise 5?. If it is, we have to add 1.50 manually (as the exercise suggests), not using the + operator.